gravitee-node-kubernetes/src/main/java/io/gravitee/node/kubernetes/keystoreloader/KubernetesConfigMapKeyStoreLoader.java

```diff
@@ -75,7 +79,10 @@ public static boolean canHandle(KeyStoreLoaderOptions options) {
             kubernetesLocations != null &&
             !kubernetesLocations.isEmpty() &&
             SUPPORTED_TYPES.contains(options.getKeyStoreType().toLowerCase()) &&
-            kubernetesLocations.stream().allMatch(location -> CONFIGMAP_PATTERN.matcher(location).matches())
+            (
+                kubernetesLocations.stream().allMatch(location -> CONFIGMAP_PATTERN.matcher(location).matches()) ||
+                kubernetesLocations.stream().allMatch(location -> PEM_REGISTRY_CONFIGMAP_PATTERN.matcher(location).matches())
```

Comment:
Is it expected to avoid mixing configmap and pem registry configmap?
Should it be

```java
kubernetesLocations.stream().allMatch(location -> CONFIGMAP_PATTERN.matcher(location).matches() ||
       PEM_REGISTRY_CONFIGMAP_PATTERN.matcher(location).matches())
```

Instead?

Reply:
You marked it as resolved but I don't see any change or comment so I unresolve it.